core/modules/wiki-bulkops.js

```diff
@@ -13,10 +13,38 @@ Bulk tiddler operations such as rename.
 "use strict";
 
 /*
+Retarget tags and lists from fromTitle toTitle.
+*/
+exports.retargetReferences = function(fromTitle,toTitle) {
```

Comment:
I went back and forth on that, but yeah, seems like a good idea and I'll add it.